pageserver/src/tenant.rs

```diff
@@ -2503,7 +2503,7 @@ impl Tenant {
             }
         }
 
-        info!("persisting tenantconf to {config_path}");
+        debug!("persisting tenantconf to {config_path}");
```

Comment:
Ah finally, I've been so annoyed by this, but thought it changed a bit with the generations so have been keeping quiet.

Excellent that we can finally let go, logging full paths is an bad idea anyways.

Reply:
Hmm ok there's a consensus that this is rare, we must have a duplicate then somewhere.

Reply:
> logging full paths is an bad idea anyways.

I disagree, it was helpful for me during debugging. Same goes for full URLs. You don't have to manually reconstruct the path but can actually directly copy it from the log and then inspect the file should the need arise.

Reply:
In the context of tests, it can quickly give you the path so you don't need to combine test_output directory, but in the prod/staging logs they are just noise as we have just one path for everything.

Reply:
I don't understand why do you think this is rare, staging is full of them:

```
2023-12-05T13:30:51.263652Z  INFO init_tenant_mgr:persist_tenant_config{tenant_id=3717abd2d0bd5266f9d031b24b6cf9fe shard_id=0000}:persist_tenant_config_at{tenant_id=3717abd2d0bd5266f9d031b24b6cf9fe shard_id=0000}: persisting tenantconf to /storage/pageserver/data/tenants/3717abd2d0bd5266f9d031b24b6cf9fe/config-v1
2023-12-05T13:30:51.265080Z  INFO init_tenant_mgr:persist_tenant_config{tenant_id=735fc681d2cc55acdc89a9c43882f60c shard_id=0000}:persist_tenant_config_at{tenant_id=735fc681d2cc55acdc89a9c43882f60c shard_id=0000}:persist_tenant_config_legacy{tenant_id=735fc681d2cc55acdc89a9c43882f60c shard_id=0000}: persisting tenantconf to /storage/pageserver/data/tenants/735fc681d2cc55acdc89a9c43882f60c/config
```

So is prod:

```
2023-12-05T13:17:32.381120Z  INFO request{method=POST path=/v1/tenant/ request_id=2eb7d9745f22b893cb6223686598df26}:tenant_create{tenant_id=X}:persist_tenant_config_at{tenant_id=Y}:persist_tenant_config_legacy{tenant_id=X}: persisting tenantconf to /storage/pageserver/data/tenants/c265ae5f7c0040ad226f0ebdc693b38f.___temp/config
```